spec/features/indexing_custom_component_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe 'Indexing Custom Component', type: :feature do
+  let(:components) do # an Array<Arclight::CustomComponent>
+    options = { component: Arclight::CustomComponent }
+    ENV['SOLR_URL'] = 'http://127.0.0.1:8983/solr/blacklight-core'
```

Comment:
Ok, I was able to find it in `Blacklight.connection_config[:url]`. Force pushed an update for that.